packages/babel-plugin-transform-es2015-modules-systemjs/src/index.js

```diff
@@ -68,7 +72,17 @@ export default function ({ types: t }) {
   };
 
   return {
+    inherits: syntax,
```

Comment:
Strictly speaking yes. But the System module format must be tied to this import syntax, so the System format strictly speaking isn't a purely es2015 feature at this point.

This import syntax is somewhat of an exception though as it is such a low-level language feature.

If we had the ability to have a plugin option determine this syntax inheritance that could work, but I don't believe that is currently possible in Babel?